bus-board/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import {getDataForStopPoints} from './apiClients/BusStopPointAPI'
+import {IJsonBus} from "./modules/IJsonBus";
+import {BusDataTable} from "./components/BusDataTable";
+
+async function getBuses(stopPoint: string): Promise<IJsonBus[]> {
+    const bussesArray: IJsonBus[] | undefined = await getDataForStopPoints(stopPoint);
+    return bussesArray != undefined ? bussesArray : new Array<IJsonBus>();
```

Comment:
In most cases we want to use `!==` operator. More about relational operators in typescript [here](https://reflectoring.io/typescript-operators/)

Reply:
Instead of returning an empty array, let's treat this error case in the component. Maybe display something about the error.